### PR TITLE
chore: add doc on pixel perfection

### PR DESCRIPTION
**Pull request by J-Son89**
A small doc we can use when creating issues, particularly for external contributors to help them have the correct knowledge and expectations before starting a task.

### CODE REVIEW

**Review by jo-mut — APPROVED** [review]
Nice

**Review by siddarthkay — APPROVED** [review]

**Comment by vkjr** [comment]
@J-Son89, could you please add the link to the doc into `doc/README.md`, so it is easier to spot?

**Review by vkjr — APPROVED** [review]

**Comment by J-Son89** [comment]
> @J-Son89, could you please add the link to the doc into `doc/README.md`, so it is easier to spot?

For sure, thanks!

**Review by ajayesivan — APPROVED** [review]
good one

File: doc/pixel-perfection.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+The Status Mobile team aims to align the design implementation perfectly according to a given screens respective Figma design spec.
+
+Currently the Figma Designs match up with an "iPhone 11 Pro" screen size. 
+
+To test your implementation is correct you can take the following steps
+
+- Open a new Figma file so that you have write privileges. 
+- Copy in the component or screen from the Figma file.
+
+- Using the "iPhone 11 Pro" simulator (it is the default now) you take a screen shot of your component and paste it into your Figma file.
```
Comment (briansztamfater) [inline doc/pixel-perfection.md]: screenshot*

**Review by briansztamfater — APPROVED** [review]

**Pull request merged**